app/vector_icons/BUILD.gn

```diff
@@ -10,28 +10,37 @@ import("//components/vector_icons/vector_icons.gni")
 aggregate_vector_icons("brave_vector_icons") {
   icon_directory = "."
 
-  sources = [
-    "bat.icon",
-    "brave_ads_close_button.icon",
-    "brave_ads_dark_mode_info_button.icon",
-    "brave_ads_light_mode_info_button.icon",
-    "brave_news_arrow_right.icon",
-    "brave_news_follow_button.icon",
-    "brave_news_subscribe.icon",
-    "brave_news_unfollow_button.icon",
-    "brave_sad.icon",
-    "brave_tooltips_stopwatch.icon",
-    "brave_translate.icon",
-    "download_unlock.icon",
-    "open_in_ipfs.icon",
-    "open_in_tor.icon",
-    "sidebar_toolbar_button.icon",
-    "sidebar_toolbar_button_right.icon",
-    "vpn_indicator.icon",
-    "wallet_toolbar_button.icon",
-    "web_discovery_infobar_close_button.icon",
+  # The Skia icon names are transformed to make them easier to use in Brave Core
+  # `arrow-right.svg` becomes `leo_arrow_right.icon`
+  leo_icons = [
+    "leo_arrow_right.icon",
+    "leo_heart_filled.icon",
+    "leo_heart_outline.icon",
+    "leo_rss.icon",
   ]
 
+  # Note: Before adding another icon here, check if the icon is available in Leo
+  # https://leo.bravesoftware.com/?path=/story/icon--all-icons
+  sources =
+      [
+        "bat.icon",
+        "brave_ads_close_button.icon",
+        "brave_ads_dark_mode_info_button.icon",
+        "brave_ads_light_mode_info_button.icon",
+        "brave_sad.icon",
+        "brave_tooltips_stopwatch.icon",
+        "brave_translate.icon",
+        "download_unlock.icon",
+        "open_in_ipfs.icon",
+        "open_in_tor.icon",
+        "sidebar_toolbar_button.icon",
+        "sidebar_toolbar_button_right.icon",
+        "vpn_indicator.icon",
+        "wallet_toolbar_button.icon",
+        "web_discovery_infobar_close_button.icon",
+      ] +
+      rebase_path(leo_icons, ".", "../../node_modules/@brave/leo/icons-skia")
```

Comment:
I think `//brave/node_modules/@brave/leo/icons-skia` is more appropriate here.

nit: wdyt about this?
```
sources = [ ... ]

leo_icons = [ ... ]
sources += rebase_path(leo_icons, ".", "//brave/node_modules/@brave/leo/icons-skia")
```

Reply:
Much neater, thanks @goodov (and it's nice not to have the weird format change)